src/calculateRentalCost.js

```diff
@@ -1,10 +1,23 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 0;
```

Comment:
Use `const` for `rentalCost` since it's not being reassigned, but rather calculated once based on the conditions.